foodsearch/src/components/pages/Indian.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import './Indian.css'
+import { Link } from "react-router-dom";
+export default function Indian(){
+    return (
+        <>
+        <div className="display-img">
+            <img src="https://restaurantindia.s3.ap-south-1.amazonaws.com/s3fs-public/content6553.jpg"></img>
```

Comment:
Try to not use hardcoded values, make an object or fake data so that later it is easy to switch to backend.